### ISSUE TITLE
Possible wrong task_name in PEMS scripts

### ISSUE THREAD

**Issue by boyufan** [kwuking/TimeMixer#55]
Hi, thank you for the excellent work. I noticed that in scripts/short_term_forecast/PEMS/TimeMixer.sh, the input value of --task_name is "long_term_forecast", shouldn't it be "short_term_forecast"?

**Comment by kwuking** [comment]
Thank you very much for your attention to our work and for your invaluable suggestions. The reason we incorporated "long_term_forecast" into the pems scripts is due to the reuse of the pipeline process from exp_long_term_forecast. It is worth noting that the [TSLib](https://github.com/thuml/Time-Series-Library) framework we are currently utilizing has the "exp_short_term_forecast," which is tailored specifically for the M4 dataset. However, given its specifics, it may not be the most suitable option for the Pems dataset. As a result, we opted for "long_term_forecast." Once again, we appreciate your attention and valuable input.
